docs/admin/installation/hardware.rst

```diff
@@ -449,7 +449,33 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
-.. _nucc12_recommendation:
+.. _nuc13_recommendation:
+
+Intel 13th-gen NUC
```

Comment:
Should we call this the "ASUS 13th-gen NUC" since that's what the product page says? And then I suppose a small comment explaining that Intel transitioned NUCs to ASUS?

Reply:
Hmmm, you removed "Intel" from everywhere in https://github.com/freedomofpress/securedrop-docs/pull/540 so seems like the answer is yes.

Reply:
Good call @legoktm! It looks like https://github.com/freedomofpress/securedrop-docs/pull/542/commits/8c5d7017f0e37539d647958121ddd8834ab6027c took care of the name switch, and then I just tacked on https://github.com/freedomofpress/securedrop-docs/pull/542/commits/55568e2286ec486c79e432d729f536e101388dfa which should cover off the explanation about the transition to ASUS. Please let me know what you think.